conf/helmfile.d/0610.prometheus-adapter.yaml

```diff
@@ -15,184 +15,34 @@ releases:
 #   - https://github.com/helm/charts/tree/master/stable/prometheus-adapter
```

Comment:
Again, nice pruning! Functionality is still intact?

Reply:
Yep, as mentioned in another comment, all the values will inherit from the chart, we only need to include values if we want to override them. Keeping old values was an issue when upgrading the chart version.